### PR TITLE
mvcc:add ut for Revisions/CountRevisions and remove RangeSince as it …

### PR DESCRIPTION
**Pull request by wayblink**
…is not used

Signed-off-by: wayblink <wayasxxx@gmail.com>

### CODE REVIEW

File: server/storage/mvcc/index.go [hunk 1]
```diff
@@ -107,6 +105,9 @@ func (ti *treeIndex) unsafeVisit(key, end []byte, f func(ki *keyIndex) bool) {
 	})
 }
 
+// Revisions returns limited number of revisions from key(including) to end(excluding)
+// at the given rev. The returned slice is sorted in the order of key.
+// There is no limit if limit <= 0.
```
Comment (ptabor) [inline server/storage/mvcc/index.go:110]: It seems also important that 'total' is not capped by the limit and reflects all the revisions.

Comment (ahrtr) [reply server/storage/mvcc/index.go:110]: Comments:
1. `including` to `included`, and `excluding` to `excluded`?  This applies to the comment for method `CountRevisions` as well.
2. adding comment something like "The second return parameter `total` isn't capped by the `limit`, and reflects all the expected revisions".

I'd defer to @spzala to correct the syntax of the comment.

**Comment by ahrtr** [comment]
Overall looks good to me. Thank you.

Leave @spzala to final decision.

File: server/storage/mvcc/index.go (hunk 1 above)
Comment (wayblink) [reply server/storage/mvcc/index.go]: Thanks for review, update as you commented.

File: server/storage/mvcc/index.go [hunk 2]
```diff
@@ -107,6 +105,9 @@ func (ti *treeIndex) unsafeVisit(key, end []byte, f func(ki *keyIndex) bool) {
 	})
 }
 
+// Revisions returns limited number of revisions from key(included) to end(excluded)
+// at the given rev. The returned slice is sorted in the order of key. There is no limit if limit <= 0.
+// The second return parameter total isn't capped by the limit, and reflects all the expected revisions.
```
Comment (ahrtr) [inline server/storage/mvcc/index.go:110]: Minor comment: replace total with `total`.

**Review by ahrtr — APPROVED** [review]

**Comment by spzala** [comment]
> Overall looks good to me. Thank you.
> 
> Leave @spzala to final decision.

Thanks @ahrtr Looking right now.

**Review by spzala — APPROVED** [review]
One comment in-line but otherwise looks good. Thanks @wayblink

File: server/storage/mvcc/index.go (hunk 2 above)
Comment (ahrtr) [reply server/storage/mvcc/index.go:110]: @wayblink  Please update per the comment. Either replace total with `total`, or just remove the parameter name per @spzala 's comment. 

I will merge this PR once it's updated and pipeline is green.

Comment (wayblink) [reply server/storage/mvcc/index.go]: Updated. Thanks a lot

**Pull request merged**